tests/test_utils.py

```diff
@@ -17,3 +35,80 @@ def test_rst_to_html():
         "</p></main>"
     )
     assert rst_to_html(really_bad_string).replace("\n", "") == expected_output
+
+
+def test_plaintext_to_html_basic_formatting():
+    text = "This is *italic* and **bold** text.\n\nNext paragraph.\nIgnore single linebreak."
```

Comment:
For readability, consider:

```suggestion
    text = """This is *italic* and **bold** text.

Next paragraph.
Ignore single linebreak."""
```